<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hadesvm::cereon::Fdc1ControllerEditor</class>
 <widget class="QWidget" name="hadesvm::cereon::Fdc1ControllerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="portsLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ports (hex):</string>
   </property>
  </widget>
  <widget class="QLabel" name="interruptMaskPortLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>INTERRUPT_MASK:</string>
   </property>
  </widget>
  <widget class="QLabel" name="stateAndControlPortLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>STATE/COMMAND:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stateAndControlPortLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="clockNumberOfUnitsLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="clockLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clock:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataPortLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DATA:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataPortLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="interruptMaskPortLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="clockUnitComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>stateAndControlPortLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hadesvm::cereon::Fdc1ControllerEditor</receiver>
   <slot>_onStateAndControlPortLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataPortLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hadesvm::cereon::Fdc1ControllerEditor</receiver>
   <slot>_onDataPortLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interruptMaskPortLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hadesvm::cereon::Fdc1ControllerEditor</receiver>
   <slot>_onInterruptMaskPortLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clockNumberOfUnitsLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hadesvm::cereon::Fdc1ControllerEditor</receiver>
   <slot>_onClockNumberOfUnitsLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clockUnitComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>hadesvm::cereon::Fdc1ControllerEditor</receiver>
   <slot>_onClockUnitComboBoxCurrentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_onStateAndControlPortLineEditTextChanged(QString)</slot>
  <slot>_onDataPortLineEditTextChanged(QString)</slot>
  <slot>_onInterruptMaskPortLineEditTextChanged(QString)</slot>
  <slot>_onClockNumberOfUnitsLineEditTextChanged(QString)</slot>
  <slot>_onClockUnitComboBoxCurrentIndexChanged(int)</slot>
 </slots>
</ui>
